src/appforms/src/core/020-utils02webcam.js

```diff
@@ -24,10 +24,18 @@ appForm.utils = function (module) {
     checkEnv();
     _html5Camera(params, cb);
   }
+
   function cancelHtml5Camera() {
-    if (localMediaStream) {
-      localMediaStream.stop();
-      localMediaStream = null;
+    if (localMediaStream){
+      if (localMediaStream.stop) {
+        localMediaStream.stop();
+        localMediaStream = null;
+      } else{
+        var tracks = localMediaStream.getTracks();
```

Comment:
undefined exception will appear only when using non defined variable. In this case localMediaStream is defined in line 14 so there is no need to check this.